README.md

```diff
@@ -47,10 +47,12 @@ specification to describe this ability as well, and move the document to the sta
   * todo
 
 ## Goals
-  * todo
+* Enable a web app to run tasks periodically on network connectivity.
 
 ## Non-goals
-  * todo
+* Triggering events at a specific time is an explicit non-goal. The [Scheduled Task API](https://bugs.chromium.org/p/chromium/issues/detail?id=889077) enables that.
```

Comment:
Please refer to "A more generalized alarms API" rather than the Chromium bug - it makes it sound like that's an official source, which it's not.

Reply:
ok, made the text super generic and removed the bug reference.